dev/airflow.Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM apache/airflow:latest
```

Comment:
I got those warnings too, but it still worked for me. I just pinned it though because it's misleading

Reply:
It could be that it worked and I was initially having the other issue where the host (localhost  vs airflow) was wrong in my .env files.  I know the update pins it to a latest as well but should work for now.